src/components/pool/branded-redirect/constants.ts

```diff
@@ -14,4 +14,11 @@ export const BRANDED_REDIRECT_DATA: Record<string, BrandedRedirectData> = {
     btnText: 'brandedRedirect.xave.btnText',
     link: 'https://app.xave.co/',
   },
+  gyro: {
+    id: 'xave',
+    title: 'brandedRedirect.gyro.title',
+    description: 'brandedRedirect.gyro.description',
+    btnText: 'brandedRedirect.gyro.btnText',
+    link: 'https://app.gyro.finance/',
```

Comment:
These point to locales in default.json. If you search for `brandedRedirect` in that file you should find the xave one and be able to add a new object for gyro.